package.json

```diff
@@ -73,17 +73,18 @@
     "eslint-config-vkbansal": "~5.2.1",
     "eslint-import-resolver-webpack": "~0.13.2",
     "eslint-plugin-import": "~2.26.0",
-    "eslint-plugin-jest": "^26.1.4",
+    "eslint-plugin-jest": "^26.1.5",
     "eslint-plugin-jest-dom": "^4.0.1",
     "eslint-plugin-react": "~7.29.4",
-    "eslint-plugin-testing-library": "^5.2.1",
+    "eslint-plugin-testing-library": "^5.3.1",
     "history": "~5.3.0",
     "html-webpack-plugin": "~5.5.0",
-    "jest": "~27.5.1",
+    "jest": "~28.0.3",
+    "jest-environment-jsdom": "^28.1.0",
```

Comment:
By using version 28.1.0 you also imported some jest packages at version 28.1.0 instead of 28.0.3. I think you should use the same version as the rest of the jest packages. Would you update it?